git/diff.py

```diff
@@ -49,6 +49,7 @@ def diff(self, other=Index, paths=None, create_patch=False, **kwargs):
             If None, we will be compared to the working tree.
             If Treeish, it will be compared against the respective tree
             If Index ( type ), it will be compared against the index.
+            If the string 'root', it will compare the empty tree against this tree.
```

Comment:
When using the string 'root', I am afraid that it would permanently block usage of a similarly named *treeish*, which could be the name of a tag or ref for example.
How would you circumvent this problem (or is it maybe not a problem at all) ?